src/utils.ts

```diff
@@ -444,3 +446,38 @@ export function applyPathMapping(
 
     return sourcePath;
 }
+
+/**
+ * Gets the localized resources that will be displayed in the frontend
+ * @param extensionPath The root folder for the extension path.
+ */
+export async function getLocalizedStrings(extensionPath: string): Promise<string> {

```

Comment:
Missing tests for this function?

Reply:
this one in specific is hard to test as it mostly calls vscode native methods (to get the language and then to load the file), inside the test code it is always mocked.